### PR TITLE
docs(certificates): New certificate reference docs

### PR DESCRIPTION
**Pull request by Kimberly-Davis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

mkdocs.yml
\docs\Reference\Certificates\IEEE8021x.md
\docs\Reference\Certificates\remoteProvisioning.md
\docs\Reference\Certificates\tls.md
mkdocs.yml

### CODE REVIEW

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+--8<-- "References/abbreviations.md"
+	
+## What is remote provisioning?
+
+Provisioning is the overall process of bringing a remote device under the management of the development system. The process involves activation and configuration of a device without physically touching it.
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:5]: I think we should move away from using development system in these docs.  Keep it to more full deployment friendly terms like:

Provisioning is the overall process of bringing a remote device under the management **of the Management Presence Server (MPS).**

There's a few references to dev system throughout.

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+!!! Note Whatever 
+    Before provisioning, identify the environment in which the device will be managed. The environments include Public Cloud, Private Cloud, and Hybrid Cloud.
+
+The table below describes the process: 
+
+| Step       | Significance to Provisioning|  
+| :------------------------- | :--------------------- | 
+| 1. Creating a Client Initiated Remote Access (CIRA) configuration | This saved configuration will be used to configure the CIRA protocol for public cloud environments. |
+| 2. Setting up the AMT Profile and choosing an activation type | Choose from two modes, Admin Control Mode (ACM) and Client Control Mode (CCM). If ACM is chosen, add a provisioning certificate in Step 3. CCM does not require a certificate. |
+| 3. Adding a domain profile and associated provisioning certificate (signed or self-signed) | Upload a provisioning certificate. |
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:16]: @matt-primrose Is self-signed an option here? Can you act as your own CA and AMT is ok with it?

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+| 4. Using the rpc-go client application to activate the remote device | Deploy and run the client as Administrator on the desired remote platform. Leverage the profile you created in Step 2.  |
+
+
+Figure 1 illustrates the steps below. Also, see Get Started
+
+To manage a device remotely, you must *provision* it, securely configure it to communicate with the development system, the provisioning server. The toolkit accomplishes secure provisioning with a process of authentication and a tunnel using Transport Layer Security (TLS) protocol.
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:22]: The first sentence reads strange and is a little inaccurate.  Something like below.  Still don't know how to incorporate the 'securely configure it' into something more grammatically easy to read.

To manage a device remotely, you must *provision* it, securely configure it **using the Remote Provisioning Server** to communicate with the **Management Presence Server**~~, the provisioning server~~.

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+
+Figure 1 illustrates the steps below. Also, see Get Started
+
+To manage a device remotely, you must *provision* it, securely configure it to communicate with the development system, the provisioning server. The toolkit accomplishes secure provisioning with a process of authentication and a tunnel using Transport Layer Security (TLS) protocol. 
+
+This authentication process involves:
+
+* **A provisioning certificate*:** an electronic document used to establish the identity of the holder
+* **CIRA:** a protocol used by the remote device to "call home", initiate remote connectivity, with a remote provisioning server.
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:27]: CIRA maintains connection to a MPS, not any prov server.  Provisioning communication is all through RPC only when actively running a cmd.

a protocol used by the remote device to "call home", initiate remote connectivity, with a Management Presence Server ~~remote provisioning server~~.

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+After a device is provisioned, the administrator can use the development system to perform different control activities, such as [power actions (e.g., Reset)](../powerstates.md) without an onsite visit.
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:29]: ...the administrator can use a ~~development system~~ **management console** to perform...

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+## What about manual provisioning?
+
+Sometimes it's not possible to deploy the rpc-go client application to activate the device remotely. When remote provisioning is not an option, the provisioning process may involve physically touching the devices. 
+
+To provision manually for ACM, see [Set a DNS Suffix through MEBX](../MEBX/dnsSuffix.md).
+
+## What is a remote provisioning certificate?
+
+A remote provisioning certificate is used for the activation of the remote device by the provisioning server, the development system. The activation phase of provisioning *turns on* or enables Intel AMT in the remote device. The Intel AMT firmware contains the root certificate hashes, TLS-based thumbprints, of participating vendors, known as certificate authorities (CA). The root certificate hash is compared with that certificate's hash. If they match, the certificate is deemed valid.
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:44]: development system

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+### Purchase
+
+To use a provisioning certificate in the toolkit, purchase the certificate from a CA. 
+
+!!! Note
+    * You do not have to purchase a certificate for each device in a fleet. The certificate is purchased for the provisioning server. 
+    * Prices vary per vendor.
+
+To get a list of participating vendors, see [Intel Developer Zone](https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html).
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:60]: We can jump them right to it by appending #linklistblade-3

i.e. https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html#linklistblade-3

File: docs/Reference/Certificates/tls.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+## What is a TLS digital certificate?
+
+Electronic documents, called certificates, are issued by and purchased from Certificate Authorities (CA), organizations that comply with TLS industry regulations, requirements, and standards.
+
+When a client running a web application, such as a browser, connects to a server, the server sends a TLS certificate that can be used to verify the authenticity of a server. The web application uses certificates present on the client to establish a secure connection. 
+
+There are two ways to use TLS in the toolkit:
+
+* Purchase a certificate from a CA and inject it via a proxy application 
+* Generate a self-signed certificate with mesh commander
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/tls.md:20]: **MeshCommander**

Maybe link them to it too? https://www.meshcommander.com/meshcommander

File: docs/Reference/Certificates/tls.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+## What is a self-signed certificate?
+
+In addition to purchasing a certificate from a CA, it is possible to generate free self-signed certificates. Self-signed certificates offer trust value within an organization and are appropriate for development, but they are not appropriate for securing content that is presented to outside world.
+
+## How does TLS work in the Toolkit?
+
+Intel® AMT is preloaded with TLS-based certificate thumbprints, a string of hexadecimal characters that identifies a certificate, of participating vendors. When you purchase and install a certificate from a participating vendor, you'll be able to use the toolkit to perform remote configuration. 
+
+To get a list of participating vendors, see [Intel Developer Zone](https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html).
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/tls.md:40]: We can jump them right to it by appending #linklistblade-3

i.e. https://www.intel.com/content/www/us/en/developer/topic-technology/edge-5g/tools/open-amt-cloud-toolkit.html#linklistblade-3

**Review by bwendlandt-intel — CHANGES_REQUESTED** [review]

File: docs/Reference/Certificates/remoteProvisioning.md (hunk 1 above)
Comment (bwendlandt-intel) [reply docs/Reference/Certificates/remoteProvisioning.md:5]: Or maybe not MPS, maybe 'of a Management Console Solution' or something. I'll let you wordcraft and decide what's best based on what you are targeting for this doc.

File: docs/Reference/Certificates/remoteProvisioning.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
+--8<-- "References/abbreviations.md"
+	
+## What is remote provisioning?
+
+Provisioning is the overall process of bringing a remote device under the management of the development system. The process involves activation and configuration of a device without physically touching it. 
+
+!!! Note Whatever
```
Comment (bwendlandt-intel) [inline docs/Reference/Certificates/remoteProvisioning.md:7]: Just noticed this hahaha, probaaaaably should go

File: docs/Reference/MEBX/unprovision.md [hunk 11]
```diff
@@ -1,23 +1,30 @@
 ## Manageability Engine BIOS Extensions (MEBX)
 
-Intel® MEBX allows for configuration of the Intel Manageability Engine (ME) platform. Through this interface, you can provision AMT and customize a variety of settings manually.
+Intel® MEBX allows for configuration of the Intel Manageability Engine (ME) platform. Use this interface to provision and unprovision AMT. It also provides a variety of settings to configure manually. 
+
+Use the unprovision functionality remove a device from control MPS control.
```
[the remark this answers is not in the document]
Comment (Kimberly-Davis) [reply docs/Reference/MEBX/unprovision.md]: Ugh. That sentence got checked in prematurely and disastrously. Changed.

File: docs/Reference/Certificates/remoteProvisioning.md (hunk 2 above)
Comment (matt-primrose) [reply docs/Reference/Certificates/remoteProvisioning.md]: CIRA certificate is always self-signed.  There is an option for a self-signed provisioning certificate, however, it requires someone injecting the root hash into the FW either at the OEM factory or through MEBX.  Not something most folks do.

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**